### PR TITLE
Biotin ED tutorial

### PR DESCRIPTION
**Pull request by dagewa**
A new tutorial, provided by Jessica Bruhn at NanoImaging Services, on using DIALS for small molecule 3DED

### CODE REVIEW

File: doc/sphinx/documentation/tutorials/3DED/Biotin.rst [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+
+Note about pedestal and offset
+==============================
+
+* Data collected with the CETA/CETA-D cameras includes some pixels for
+  which the recorded intensity value is negative. Many of these pixels
+  likely still contain valuable information and therefore should not be
+  entirely excluded from the data.
+* SMV format, a common format for X-ray crystallography diffraction images,
+  does not allow for “unsigned data” (negative values). Because
```
Comment (biochem-fan) [inline doc/sphinx/documentation/tutorials/3DED/Biotin.rst:47]: Not _unsigned_ but _signed_?

File: doc/sphinx/documentation/tutorials/3DED/Biotin.rst [hunk 2]
```diff
@@ -0,0 +1,696 @@
...
+  of this, Leginon adds an offset value to all pixels when converting to
+  SMV format to make all values positive. Leginon also excludes negative
+  values beyond a reasonable threshold.
+* Some data processing programs, such as DIALS, allow for unsigned
```
Comment (biochem-fan) [inline doc/sphinx/documentation/tutorials/3DED/Biotin.rst:51]: allow for _signed_ data?

File: doc/sphinx/documentation/tutorials/3DED/Biotin.rst [hunk 3]
```diff
@@ -0,0 +1,696 @@
...
+  data. Therefore, it can be beneficial to add back in a reasonable
```
Comment (biochem-fan) [inline doc/sphinx/documentation/tutorials/3DED/Biotin.rst:52]: _subtract_ not _add back_?

**Review by biochem-fan — COMMENTED** [review]
There are some weird statements about pedestal offsets.

**Comment by dagewa** [comment]
@biochem-fan yes, thanks for the suggestions. I wanted to get the tutorial built so I could share easily with Jessica to see if she was happy with the result, hence the quick merge. I will include your and her suggestions in a commit to update.

**Pull request merged**